### ISSUE TITLE
Allow users to enable IPv6

### ISSUE THREAD

**Issue by MysticRyuujin** [pia-foss/desktop#1]
**Is your feature request related to a problem? Please describe.**
IPv6 no longer functions on local network.

**Describe the solution you'd like**
Please re-enable the previously available option to **NOT** disable IPv6. I'm a big boy, I understand the risks. Give me the option.

**Describe alternatives you've considered**
Potentially allow IPv6 whitelisting instead. My internal network is a known IPv6 range. I would like to be able to communicate over my local LAN at the very least. Blackhole everything else?

Alternatively, support IPv6 in full

**Additional context**
N/A

**Comment by JonathonH-PIA** [comment]
Hey @MysticRyuujin, we're looking at making some improvements for this to solve this without enabling IPv6 across the board since it often causes leaks (sounds like you probably have this covered, but that's not always the case).

We're looking at expanding our Allow LAN feature to also permit your local IPv6 /64 subnet.  Right now we allow all private use IPv4 ranges as well as link-local IPv6 addresses, but we really should also allow the local IPv6 subnet too, and that shouldn't cause leaks.  Would that work for your use case?

We're also looking into allowing manual whitelisting of IPv4/IPv6 subnets as part of our split tunnel feature, but this is probably a bit farther off in our timeline.

**Comment by MysticRyuujin** [comment]
The only problem with only allowing the local subnet is if you have a /48 (like I do) that's been sub divided into /64 subnets (like I have) you'd still be blocking communication across my LAN subnets. If the mask was configurable, that'd work.

**Comment by MysticRyuujin** [comment]
I wanted to leave another quick update here.

I recently switched my PIA Desktop app default to "Bypass VPN" and then selected "VPN Only" on the apps I wanted on the VPN. I did this mostly because of the usability of Windows Store and Microsoft Office.

Anyway, even when in this mode PIA disabled IPv6 across the board. Not sure if this is necessary in this mode or not. Not that I think it's necessary in general, but still.
